includes/Experiments.php

```diff
@@ -313,6 +313,17 @@ public function get_experiments(): array {
 				'description' => __( 'Allow locked stories to be taken over by another authors', 'web-stories' ),
 				'group'       => 'editor',
 			],
+			/**
+			 * Author: @brittanyirl
+			 * Issue: #10115
+			 * Creation date: 2022-02-02
+			 */
+			[
+				'name'        => 'enableUpdatedPublishStoryModal',
+				'label'       => __( 'Updated Publish Story Modal', 'web-stories' ),
+				'description' => __( 'Enable new modal that shows up on Story publish to make sure necessary info is all present before publishing', 'web-stories' ),
```

Comment:
That's quite a long description that is hard to read and translate.

I'd aim for something simpler. For example `Enable new pre-publish confirmation modal`